osmomath/decimal_test.go

```diff
@@ -356,40 +356,40 @@ func (s *decimalTestSuite) TestDecsEqual() {
 func (s *decimalTestSuite) TestArithmetic() {
 	tests := []struct {
 		d1, d2                                osmomath.BigDec
-		expMul, expMulTruncate                osmomath.BigDec
+		expMul, expMulTruncate, expMulRoundUp osmomath.BigDec
```

Comment:
Note to reviewer: all tests in this file are copied from: https://github.com/osmosis-labs/cosmos-sdk/pull/430